nuc_morph_analysis/lib/visualization/write_data_for_colorizer.py

```diff
@@ -511,12 +510,19 @@ def make_dataset(
             )
 
             # Make the features, frame data, and manifest.
-            nframes = len(grouped_frames)
+            nframes = len(grouped_frames)         
```

Comment:
Extra whitespace at the end of a couple lines in this file? Might want to run a formatter

Reply:
Yea so weird sorry. I don't really run formatters but I could try if you think its an issue..

Reply:
@pgarrison do we have a format we follow?

Reply:
You can install a formatter in VS Code: https://code.visualstudio.com/docs/python/formatting

I use `black` - I think Saurabh talked about using black as well

Reply:
Yup, we formatted the whole repo with `black` when moving to the AllenCell org, so that would be the one to use.

Reply:
Thanks! Done!